components/nimbus/src/dbcache.rs

```diff
@@ -43,25 +44,36 @@ impl DatabaseCache {
     pub fn commit_and_update(&self, db: &Database, writer: Writer) -> Result<()> {
         // By passing in the active `writer` we read the state of enrollments
         // as written by the calling code, before it's committed to the db.
-        let experiments = get_enrollments(&db, &writer)?;
+        let enrollments = get_enrollments(&db, &writer)?;
 
         // Build the new hashmaps.  Note that this is somewhat temporary, is
         // likely to change when the full FeatureConfig stuff is implemented.
         // Further, note that, for the moment, we only (currently) support
         // one feature_id per experiment, meaning that we ignore everything
         // except the first feature_id in the array.  Some of the multi-feature
         // code may want to live in the EnrollmentEvolver.
-        let mut branches_by_experiment = HashMap::with_capacity(experiments.len());
-        let mut branches_by_feature = HashMap::with_capacity(experiments.len());
+        let mut branches_by_experiment = HashMap::with_capacity(enrollments.len());
```

Comment:
I'm wondering if this wants to be called branches_by_enrolled_experiment for clarity as to what's actually going on.

Reply:
Similarly, I suppose features_by_feature_id might want be to enrolled_features_by_feature_id

Reply:
If the user has a branch for an experiment, then they're by definition enrolled.